torch/csrc/api/include/torch/nn/options/loss.h

```diff
@@ -18,6 +18,21 @@ struct TORCH_API L1LossOptions {
 
 // ============================================================================
 
+/// Options for a KLDiv loss module.
+using KLDivLossOptions = L1LossOptions;
```

Comment:
I think we should define all the new loss options explicitly instead of aliasing `L1LossOptions`, because `L1LossOptions` could take a different default `reduction` value or get a new options arg one day, and we might not want those changes to affect the other loss options.